### PR TITLE
Enabling a textStyle prop for adding custom styles to tab bar text 

### PR DESCRIPTION
**Pull request by shidhincr**
A new prop called `textStyle` is added for handling custom styles for the tab bar text. This can be used if the developer wants to change `fontFamily` and other text styles to match with the current app theme.

Please review and provide feedback

### CODE REVIEW

File: DefaultTabBar.js [hunk 1]
```diff
@@ -31,7 +32,7 @@ const DefaultTabBar = React.createClass({
       onPress={() => this.props.goToPage(page)}
     >
       <View style={styles.tab}>
-        <Text style={{color: isTabActive ? activeTextColor : inactiveTextColor, fontWeight: isTabActive ? 'bold' : 'normal', }}>
+        <Text style={[textStyle, {color: isTabActive ? activeTextColor : inactiveTextColor, fontWeight: isTabActive ? 'bold' : 'normal', }]}>
```
Comment (skv-headless) [inline DefaultTabBar.js]: Could you please move ```textStyle``` to the end of array.

File: ScrollableTabBar.js [hunk 2]
```diff
@@ -109,9 +110,9 @@ const ScrollableTabBar = React.createClass({
       onLayout={this.measureTab.bind(this, page)}
       >
       <View>
-        <Text style={{color: isTabActive ? activeTextColor : inactiveTextColor, }}>{name}</Text>
+        <Text style={[textStyle, {color: isTabActive ? activeTextColor : inactiveTextColor}]}>{name}</Text>
```
Comment (skv-headless) [inline ScrollableTabBar.js]: Could you please move ```textStyle``` to the end of array.

File: ScrollableTabBar.js [hunk 3]
```diff
@@ -109,9 +110,9 @@ const ScrollableTabBar = React.createClass({
...
       </View>
-    </TouchableOpacity>;
+      </TouchableOpacity>;
```
Comment (skv-headless) [inline ScrollableTabBar.js]: undo this change please

File: ScrollableTabBar.js (hunk 2 above)
Comment (shidhincr) [inline ScrollableTabBar.js]: sure.. will do

File: ScrollableTabBar.js (hunk 3 above)
Comment (shidhincr) [inline ScrollableTabBar.js]: Sorry about that.. will revert.

**Comment by shidhincr** [comment]
@skv-headless Updated the PR with the changes as per comments

**Comment by shidhincr** [comment]
:+1:

**Comment by skv-headless** [comment]
thanks :+1:
